### PR TITLE
Replace neo4j integration with neo4j_enterprise integration

### PR DESCRIPTION
**Pull request by MattiasPernhult**

### CODE REVIEW

File: neo4j/CHANGELOG.md [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# CHANGELOG - Neo4j2
+# CHANGELOG - neo4j
```
Comment (Bondza) [inline neo4j/CHANGELOG.md]: Do we write the name in all lower case?

File: neo4j/datadog_checks/neo4j/__init__.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
 from .__about__ import __version__
-from .neo4j import Neo4jCheck
+from .neo4j import GLOBAL_DB_NAME, NAMESPACE, Config, Neo4jCheck
 
 __all__ = ['__version__', 'Neo4jCheck']
```
Comment (Bondza) [inline neo4j/datadog_checks/neo4j/__init__.py]: Should the new imports be included in `__all__`?

File: neo4j/tests/test_neo4j.py [hunk 3]
```diff
@@ -1,40 +1,90 @@
...
+        aggregator.assert_metric(
+            name='{}.check_point.events'.format(NAMESPACE), tags=['db_name:{}'.format(GLOBAL_DB_NAME)],
+        )
+    elif instance.get('neo4j_version') == '4.0':
+        aggregator.assert_metric(
+            name='{}.page_cache.hits'.format(NAMESPACE), tags=['db_name:{}'.format(GLOBAL_DB_NAME)],
+        )
+        aggregator.assert_metric(name='{}.check_point.events'.format(NAMESPACE), tags=['db_name:neo4j'])
+        aggregator.assert_metric(name='{}.check_point.events'.format(NAMESPACE), tags=['db_name:system'])
+    else:
```
Comment (Bondza) [inline neo4j/tests/test_neo4j.py]: Should this include a test for `4.1` as well?

Comment (MattiasPernhult) [reply neo4j/tests/test_neo4j.py]: 4.1 is not public yet :)

Comment (Bondza) [reply neo4j/tests/test_neo4j.py]: Yes, but it is in the code now so should it be removed from the code too then?

Comment (MattiasPernhult) [reply neo4j/tests/test_neo4j.py]: Yeah you are right, that might need fixing before we create a PR against the upstream repository. Good find!

Comment (Bondza) [reply neo4j/tests/test_neo4j.py]: Does it make a difference if we remove it before createing a PR against the upstream repo, isn't this repo public anyway?

**Comment by MattiasPernhult** [comment]
Superseded by https://github.com/neo-technology/integrations-extras/pull/3

**Pull request closed without merging**